lens/history.go

```diff
@@ -180,25 +150,28 @@ func getSchemaHistory(
 	ctx context.Context,
 	txn datastore.Txn,
 	schemaRoot string,
-) (map[collectionID]*schemaHistoryLink, error) {
+) (map[schemaVersionID]*schemaHistoryLink, error) {
 	cols, err := description.GetCollectionsBySchemaRoot(ctx, txn, schemaRoot)
 	if err != nil {
 		return nil, err
 	}
 
-	history := map[collectionID]*schemaHistoryLink{}
+	history := map[schemaVersionID]*schemaHistoryLink{}
+	schemaVersionsByColID := map[uint32]schemaVersionID{}
 
 	for _, c := range cols {
 		col := c
```

Comment:
I have no idea, might have made sense at one point, or it might have been a brain-fart.

I'll remove it shortly before merge.

- [ ] Remove odd var copy

Reply:
I found out, it is because we want a pointer to it, and the way Go works means that without the copy `col` would always be the last col in `cols`.

Reply:
Thanks for reminding me of this. Could you add a `TODO remove when Go 1.22` comment above it please.

Reply:
Ah nice, glad that will fix it - will do

- [ ] Add too comment